.github/ dependabot.yml

```diff
@@ -0,0 +1,14 @@
+# Please see the documentation for all configuration options:
```

Comment:
This file also has the same issue with https://github.com/slackapi/python-slack-sdk/pull/1467#discussion_r1511957488

Reply:
Thanks for pointing it out, not sure how it got there